docs/en/ingest-management/fleet/fleet-settings-output-elasticsearch.asciidoc

```diff
@@ -185,6 +185,8 @@ For descriptions of each setting, refer to <<es-output-settings-yaml-config,Adva
 As mentioned, the `custom` preset allows you to input your own set of parameters for a finer tuning of performance. The following table
 is a summary of a few data points and how the resulting EPS compares to the `Balanced` setting mentioned above.
 
+These presets apply only to agents on version 8.12.0 or later.
```

Comment:
@nimarezainia: Typically we use our built-in docs versioning to communicate this information.  Meaning that this content would be only in the 8.12 branch and later branches. 
As this could be confusing to users on pre-8.12 versions who might be looking at `current` docs, I'm inclined to add this.